File: app/src/main/java/xyz/rodit/snapmod/features/conversations/NullFilter.java
package xyz.rodit.snapmod.features.conversations;

import xyz.rodit.snapmod.features.FeatureContext;

public class NullFilter extends ObjectFilter<Object> {

    public NullFilter(FeatureContext context, String configKey) {
        super(context, configKey, (Object) null);
    }

    @Override
    public boolean shouldFilter(Object object) {
        return true;
    }
}

File: app/src/main/java/xyz/rodit/snapmod/features/conversations/NullFilter.kt
package xyz.rodit.snapmod.features.conversations

import xyz.rodit.snapmod.features.FeatureContext

class NullFilter(context: FeatureContext, configKey: String) : ObjectFilter<Any?>(context, configKey, null) {

    override fun shouldFilter(item: Any?): Boolean {
        return true
    }
}